send_webhooks.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# workflow-helpers
+# Copyright(C) 2022 Kevin Postlethwait
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""This script sends all webhooks in a directory, each is represented by a JSON object."""
+
+import json
+import logging
+import os
+
+from kubernetes import kubernetes as k8
+
+import requests
+
+dir = os.getenv["WEBHOOK_DIR"]
+document_id = os.getenv["DOCUMENT_ID"]
+secret_namespace = os.getenv["THOTH_MIDDLETIER_NAMESPACE"]
```

Comment:
callback_url is response of the user-request, right? (example: adviser)
the data service for user-request stay in the backend namespace, 
can you share why we are using middletier namespace?

Reply:
Okay, will do